modules/tensor_mechanics/test/tests/global_strain/global_strain_action.i

```diff
@@ -40,6 +34,19 @@
 
 [Modules]
   [./TensorMechanics]
+    # Master action for generating the tensor mechanics kernels, variables,
+    # strain calculation material, and the auxilliary system for visualization
+    [./Master]
+      [./stress_div]
+        strain = SMALL
+        add_variables = true
+        global_strain = global_strain
```

Comment:
Please update the documentation page for the Tensor Mechanics master action to include a brief description and a link to the documentation page for global strain in both the table and the bulleted list.  Having some way to explain the difference between the global strain and the finite strain that we use in larger deformation simulations will be important--many of our engineering scale mechanics users could be confused by the global strain option

Reply:
This does not really add any new functionality of the master action. It just provides the input to the strain calculators (think additional strain contribution similar to eigen strain). I am not sure it's worth mentioning in the description or the table. I guess I should rather add a more informative description of the input variable.

Reply:
Please do add the input parameter to the table along with the link to your class description.  Thanks for commenting about the eigenstrains--I see that I have not included the eigenstrains in the table either, so I will make a merge request to add the eigenstrains to the master action table too.  Thanks!